src/main/java/mate/controller/GetAllCarsWithDrivers.java

```diff
@@ -0,0 +1,26 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+@WebServlet(urlPatterns = "/cars/drivers")
+public class GetAllCarsWithDrivers extends HttpServlet {
```

Comment:
you already have GetAllCarsController. Why do you need this one? They are identical